airflow/docker.go

```diff
@@ -107,30 +107,50 @@ func Start(airflowHome string) error {
 	projectName := config.GetString(config.CFGProjectName)
 
 	// Create a libcompose project
-	project, err := createProjectFromContext(projectName, airflowHome)
+	project, err := fetchDockerProject(projectName, airflowHome)
 	if err != nil {
 		return errors.Wrap(err, "Error creating docker-compose project")
 	}
 
-	// Build this project image
-	imageBuild(airflowHome, imageName(projectName, "latest"))
-
-	// Start up our project
-	err = project.Up(context.Background(), options.Up{})
+	// Fetch project containers
+	psInfo, err := project.Ps(context.Background())
 	if err != nil {
-		return errors.Wrap(err, "Error building, (re)creating or starting project containers")
+		return errors.Wrap(err, "Error checking docker-compose status")
+	}
+
+	if len(psInfo) > 0 {
+		// Ensure project is not alreasdy running
+		for _, info := range psInfo {
```

Comment:
Prevents a user from running start and having a hard error on an already running project.